includes/class-wc-payment-token-sepa.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Class WC_Payment_Token_SEPA
+ *
+ * @package WooCommerce\Payments
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * WooCommerce Stripe SEPA Direct Debit Payment Token.
+ *
+ * Representation of a payment token for SEPA.
+ *
+ * @class    WC_Payment_Token_SEPA
+ */
+class WC_Payment_Token_SEPA extends WC_Payment_Token {
```

Comment:
The token classname is defaulted to prefix with `WC_Payment_Token_` from [Woo core](https://github.com/woocommerce/woocommerce/blob/trunk/includes/class-wc-payment-tokens.php#L232). 

However, this classname also exist in another plugin https://github.com/woocommerce/woocommerce-gateway-stripe/blob/trunk/includes/class-wc-stripe-sepa-payment-token.php#L18 which may cause compatibility issues if it is a concern.

Reply:
Added an issue here: https://github.com/Automattic/woocommerce-payments/issues/1605